File: HorseRacing.Contracts/Models/Game/Responses/PlayGame/PlayGameResponseDto.cs
using HorseRacing.Contracts.Base.Dto;
using HorseRacing.Contracts.Models.Game.Dtos;
using HorseRacing.Domain.GameAggregate.ValueObjects;

namespace HorseRacing.Contracts.Models.Game.Responses.PlayGame
{
    public class PlayGameResponseDto : BaseDto
    {
        /// <summary>
        /// Код игры
        /// </summary>
        public GameId GameId { get; set; }
        /// <summary>
        /// Назване игры
        /// </summary>
        public string Name { get; set; }
        /// <summary>
        /// Колода до начала игры
        /// </summary>
        public List<GameDeckCardDto> InitialDeck { get; set; } = new();
        /// <summary>
        /// Информация о лошади и игроке который на неё поставил
        /// </summary>
        public List<HorseBetDto> HorseBets { get; set; } = new();
        /// <summary>
        /// Все события по шагам
        /// </summary>
        public List<GameEventDto> Events { get; set; } = new();
    }
}

File: HorseRacing.Web/src/interfaces/api/contracts/model/game/responses/play-game/play-game-response-dto.ts
import type { BaseDto } from '~/interfaces/api/contracts/base/dto/base-dto';
import type { GameDeckCardDto } from '../../dto/game-deck-card-dto';
import type { HorseBetDto } from '../../dto/horse-bet-dto';
import type { GameEventDto } from '../../dto/game-event-dto';

export interface PlayGameResponseDto extends BaseDto {
    /** Код игры */
    GameId: string;
    /** Назване игры */
    Name: string;
    /** Колода до начала игры */
    InitialDeck: GameDeckCardDto[];
    /** Информация о лошади и игроке который на неё поставил */
    HorseBets: HorseBetDto[];
    /** Все события по шагам */
    Events: GameEventDto[];
}
